torch/_inductor/ir.py

```diff
@@ -6657,6 +6657,38 @@ def codegen(self, wrapper):
         )
 
 
+class Broadcast(InPlaceCollectiveKernel):
+    def __init__(self, layout, inputs, constant_args, src):
```

Comment:
@oulgen Would you mind double checking the usage of `NoneLayout` here? Tks!

cc @wconstab

Reply:
NoneLayout and mutationoutput looks good to me